### ISSUE TITLE
Ensure ARIA regions, landmarks and HTML sections are identifiable

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#66]
## Description
When ARIA regions, landmarks or HTML5 section elements are provided, users must be able to distinguish them from other regions, landmarks or sections in the page, particularly when two or more instances of the same type are used. When such an element does not provide a descriptive label to clearly identify itself, users of screen readers may have trouble locating the correct section or understanding its purpose.

### Breadcrumb
- [x] Breadcrumb nav should have an `aria-label`

![35a7b3d5a2a1d36ef9f2746c92cfe6db3691bfa8-1641199098](https://user-images.githubusercontent.com/8166060/153067381-a43e449e-4931-4986-a8e8-475a1f65dc55.png)

Compliant Code Example:
```
<nav class="Breadcrumbs__Nav-sc-1p78jzo-2 kxcwaC" aria-label="Breadcrumb">
  <ul class="Breadcrumbs__List-sc-1p78jzo-3 bjbnKg">
    <li class="Breadcrumbs__Item-sc-1p78jzo-4 fTEQOh">
      <a href="/about">About</a>
    </li>
    <li class="Breadcrumbs__Item-sc-1p78jzo-4 fTEQOh">
      <a href="/about/supporters-and-participants" class="">Supporters and Participants</a>
    </li>
    <li class="Breadcrumbs__Item-sc-1p78jzo-4 jHzYOJ">
      <a href="/about/supporters-and-participants/science-collaborations" class="">Vera C. Rubin Observatory science collaborations</a>
    </li>
  </ul>
</nav>
```

### Pagination (e.g. News, Events, Gallery, etc.)
- [x] Pagination nav should have an `aria-label`

![32fc6dea236ca63767370359caae5ea5ebcd940c-1641288233](https://user-images.githubusercontent.com/8166060/153068099-6c77a94c-a1ff-448b-9bfb-97e1e36d91c4.png)

Compliant Code Example:
```
<nav aria-label="Pagination" class="Pagination__NavDesktop-gl7iqs-1 jlDnac">
  <div>Showing 1 to 8 of 8</div>
  <div>1</div>
  <div>Previous&nbsp;&nbsp;Next</div>
</nav>
```

## CIC Estimate
0 points

**Comment by dananjohnson** [comment]
Breadcrumb label added in https://github.com/lsst-epo/rubin-obs-client/commit/cc9c0b7c941199a029807ed9ae5df38ec6214dd2.

**Comment by dananjohnson** [comment]
Pagination label added in 09a8d4b.
